### PR TITLE
Issue #9 and 30

### PR DESCRIPTION
**Pull request by wallscheid**
Service pull request for MS

### CODE REVIEW

File: gym_electric_motor/physical_systems/electric_motors.py [hunk 1]
```diff
@@ -1386,29 +1651,76 @@ def i_in(self, state):
...
+            eps_mag(float): magnetic field angle
+            u_q_max(float): maximal strator voltage in q-system
+            u_rq_max(float): maximal rotor voltage in q-system
+
+        returns:
+            list with maximal flux in alpha,beta system
+        """
+        mp = self.motor_parameter
+        l_s = mp['l_m'] * mp['l_ssig']
+        l_r = mp['l_m'] * mp['l_rsig']
```
[the remark this answers is not in the document]
Comment (GitPascalP) [reply gym_electric_motor/physical_systems/electric_motors.py:1727]: changed type: multiplication to addtion (+)

File: gym_electric_motor/physical_systems/electric_motors.py [hunk 2]
```diff
@@ -1386,29 +1651,76 @@ def i_in(self, state):
...
+        if omega == 0:
+            psi_d_max = mp['l_m'] * self._nominal_values['i_sd']
+        else:
+            i_d, i_q = self.q_inv([self._initial_states['i_salpha'],
+                                  self._initial_states['i_sbeta']],
+                                  eps_mag)
+            psi_d_max = mp['p'] * omega * sigma * l_s * i_q + \
+                        (mp['r_s'] + mp['r_r'] * l_mr**2) + \
+                        u_q_max + \
+                        l_mr * u_rq_max
```
[the remark this answers is not in the document]
Comment (GitPascalP) [reply gym_electric_motor/physical_systems/electric_motors.py:1740]: resolved typo: i_q to i_d

**Comment by GitPascalP** [comment]
- requested changes are worked into new push
- basic tests have been added, running pytest of all tests was succesfull
- added documentationt in docstrings and updatet doc files for documentation (mostly vor external speed load)

File: gym_electric_motor/physical_systems/mechanical_loads.py [hunk 3]
```diff
@@ -1,13 +1,52 @@
...
+        { 'states'(dict): with state names and initital values
+          'interval'(array like): boundaries for each state
+                    (only for random init), shape(num states, 2)
+          'random_init'(str): 'uniform' or 'normal'
+          'random_params(tuple): mue(float), sigma(int)
+    Example initializer(dict) for constant initialization:
+        { 'states': {'omega': 16.0}}
+    Example  initializer(dict) for random initialization:
+        { 'random_init': 'normal'}
+>>>>>>> new_issue9_30
```
Comment (max-schenke) [inline gym_electric_motor/physical_systems/mechanical_loads.py:49]: Merge incomplete

File: gym_electric_motor/physical_systems/electric_motors.py [hunk 4]
```diff
@@ -12,8 +13,48 @@ class ElectricMotor:
...
+                  'interval'(array like): boundaries for each state
+                            (only for random init), shape(num states, 2)
+                  'random_init'(str): 'uniform' or 'normal'
+                  'random_params(tuple): mue(float), sigma(int)
+        Example initializer(dict) for constant initialization:
+            { 'states': {'omega': 16.0}}
+        Example  initializer(dict) for random initialization:
+            { 'random_init': 'normal'}
+
+>>>>>>> new_issue9_30
```
Comment (max-schenke) [inline gym_electric_motor/physical_systems/electric_motors.py:57]: Merge incomplete

File: tests/test_physical_systems/test_mechanical_loads.py [hunk 5]
```diff
@@ -1,12 +1,31 @@
 import pytest
 import gym_electric_motor as gem
-from gym_electric_motor.physical_systems import PolynomialStaticLoad, MechanicalLoad, ConstantSpeedLoad
+from gym_electric_motor.physical_systems import PolynomialStaticLoad, MechanicalLoad, ConstantSpeedLoad, ExternalSpeedLoad
+from gym.spaces import Box
 import numpy as np
+from scipy import signal
 import math
+from pytest_lambda import lambda_fixture
```
Comment (max-schenke) [inline tests/test_physical_systems/test_mechanical_loads.py:8]: Does this package need to be included in the requirements.txt?

Comment (GitPascalP) [reply tests/test_physical_systems/test_mechanical_loads.py:8]: Yes (required at least for pytest scripts)

**Pull request merged**